pkg/sql/crdb_internal.go

```diff
@@ -6233,20 +6233,60 @@ func populateClusterLocksWithFilter(
 var crdbInternalNodeExecutionInsightsTable = virtualSchemaTable{
 	schema: `
 CREATE TABLE crdb_internal.node_execution_insights (
-	session_id               STRING NOT NULL,
-	transaction_id           UUID NOT NULL,
-	statement_id             STRING NOT NULL,
-	statement_fingerprint_id BYTES NOT NULL
+	session_id                 STRING NOT NULL,
+	txn_id                     UUID NOT NULL,
+	txn_fingerprint_id         BYTES NOT NULL,
+	statement_id               STRING NOT NULL,
```

Comment:
Let's abbreviate statement -> stmt

statement_id -> stmt_id
stmt_fingerprint_id -> stmt_fingerprint_id

Seems like that's the pattern: https://www.cockroachlabs.com/docs/stable/crdb-internal.html#statement_statistics